src/types/plugin-inputs.ts

```diff
@@ -18,7 +18,12 @@ export interface PluginInputs<T extends SupportedEventsU = SupportedEventsU, TU
  * The kernel will extract those and pass them to the plugin,
  * which are built into the context object from setup().
  */
-export const pluginSettingsSchema = T.Object({});
+
+export const pluginSettingsSchema = T.Object({
```

Comment:
Do we need optional with the defaults? Also be careful not sure if it works if you send an empty string to the OpenAI constructor for the base url if it works.

Reply:
I referenced [conversation rewards](https://github.com/ubiquity-os-marketplace/conversation-rewards/blob/04fba4542d4cc11444f1463cd124038cddf248d6/src/configuration/content-evaluator-config.ts#L13) and it uses an empty string so I think it's safe. 

I'm not 100% sure but is the preference to not use the `T.Optional` schema?

Reply:
Because a check is done https://github.com/ubiquity-os-marketplace/conversation-rewards/blob/2da6b868c9700b68e5bbedfb64683e42b384755c/src/parser/content-evaluator-module.ts#L29

It's fine to use optional, but usually not useful if there is a default because a value will be given either way.